modules/monitoring-adding-a-secret-to-the-alertmanager-configuration.adoc

```diff
@@ -2,132 +2,103 @@
 //
 // * observability/monitoring/configuring-the-monitoring-stack.adoc
 
-:_mod-docs-content-type: PROCEDURE
-[id="monitoring-adding-a-secret-to-the-alertmanager-configuration_{context}"]
-= Adding a secret to the Alertmanager configuration
+// The ultimate solution DOES NOT NEED separate IDs, it is just needed for now so that the tests will not break
 
-ifndef::openshift-dedicated,openshift-rosa[]
-You can add secrets to the Alertmanager configuration for core platform monitoring components by editing the `cluster-monitoring-config` config map in the `openshift-monitoring` project.
-endif::openshift-dedicated,openshift-rosa[]
-ifdef::openshift-dedicated,openshift-rosa[]
-You can add secrets to the Alertmanager configuration for user-defined projects by editing the `user-workload-monitoring-config` config map in the `openshift-user-workload-monitoring` project.
-endif::openshift-dedicated,openshift-rosa[]
+// tag::CPM[]
+[id="monitoring-adding-a-secret-to-the-alertmanager-configuration-cpm_{context}"]
+= Adding a secret to the Alertmanager configuration for core platform monitoring
+// end::CPM[]
+
+// tag::UWM[]
+[id="monitoring-adding-a-secret-to-the-alertmanager-configuration-uwm_{context}"]
+= Adding a secret to the Alertmanager configuration for monitoring of user-defined projects
+// end::UWM[]
+
+// Set attributes to distinguish between cluster monitoring example (core platform monitoring - CPM) and user workload monitoring (UWM) examples
+
+// tag::CPM[]
+:configmap-name: cluster-monitoring-config
+:namespace-name: openshift-monitoring
+:component: alertmanagerMain
+// end::CPM[]
+// tag::UWM[]
+:configmap-name: user-workload-monitoring-config
+:namespace-name: openshift-user-workload-monitoring
+:component: alertmanager
+// end::UWM[]
+
+You can add secrets to the Alertmanager configuration by editing the `{configmap-name}` config map in the `{namespace-name}` project.
 
 After you add a secret to the config map, the secret is mounted as a volume at `/etc/alertmanager/secrets/<secret_name>` within the `alertmanager` container for the Alertmanager pods.
 
 .Prerequisites
 
+// tag::CPM[]
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
```

Comment:
Would it suffice to say " . . . the `cluster-admin` role" rather than ". . . the `cluster-admin` cluster role"?

Reply:
Hello @aspauldi , there is a difference between roles and cluster roles that our engineers want  separated so that it is clear to user which is which. Therefore, it needs to stay as "cluster role".